docs/section19/19-5.md

```diff
@@ -0,0 +1,42 @@
+# Section 19 - Rule 19-5      
+**Schema Version:** 0.0.24    
+**Mandatory Rule:** True      
+**Rule ID:** 19-5       
+**Rule Description:** Unmet load hours for the proposed design shall not exceed 300 (of the 8760 hours simulated).   
+**Rule Assertion:** Options are Pass/Fail/UNDETERMINED       
+**Appendix G Section:** G3.1.2.3        
+**90.1 Section Reference:** None  
+
+**Data Lookup:** None  
+
+**Evaluation Context:** Each Proposed Design RulesetModelInstance  
+
+**Applicability Checks:**  
+
+1. Applies only to the proposed design rule set model instance.  
+
+**Function Calls:**  None
+
+## Rule Logic:   
+**Applicability Check 1:**  
+- For each RulesetModelInstance in ASHRAE229.ruleset_model_instance : `for RulesetModelInstance in ASHRAE229.ruleset_model_instance:`  
+    - Check if the ruleset model instance is the proposed model instance: `If RulesetModelInstance.RulesetModelOptions2019ASHRAE901.ruleset_model_type = "PROPOSED":`  
+        - Get the heating unmet load hours for the rule set model instance: `unmet_load_hours_heating = RulesetModelInstance.output.OutputInstance.unmet_load_hours_heating`  
+        - Get the cooling unmet load hours for the rule set model instance: `unmet_load_hours_cooling = RulesetModelInstance.output.OutputInstance.unmet_load_hours_cooling`  
```

Comment:
@claperle should we be using the unmet hrs for H and C when occupied?
Similarly, do we need to clarify in the schema notes that 'unmet_load_hours' should be only for the occupied hrs?

Reply:
Let's discuss on the call. App G does not separate occupied from unoccupied unmet load hours (as far as I know) so we did not include those data elements. I was not sure why they were included in the schema.

Reply:
Per call on 1/12/2023 - as the RDS is written is good. No changes needed.